src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/AppWithUnknownLanguageResource.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using BundleTests.Helpers;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.CoreSetup.Test;
+using Xunit;
+
+namespace AppHost.Bundle.Tests
+{
+    // https://github.com/dotnet/runtime/issues/88465
+    public class AppWithUnknownLanguageResource : BundleTestBase, IClassFixture<AppWithUnknownLanguageResource.SharedTestState>
```

Comment:
Can you put this in `Microsoft.NET.HostModel.AppHost.Tests` instead? The tests in AppHost.Bundle.Tests are really around bundling/running a single-file application in various ways, which is not relevant to this test case.

You should be able to use this helper: https://github.com/dotnet/runtime/blob/6029c304dbce7565575d1f02b295b2a0664fafaa/src/installer/tests/TestUtils/TestApp.cs#L69